src/merge.c

```diff
@@ -1666,6 +1666,27 @@ static git_iterator *iterator_given_or_empty(git_iterator **empty, git_iterator
 	return *empty;
 }
 
+static int git_merge__lookup_file_favor(
+	git_repository *repo,
+	const char *path,
+	int *file_favor)
+{
+	int error = 0;
+
+	if (path) {
+		const char *value = NULL;
+		if ((error = git_attr_get(&value, repo, 0, path, "merge")) < 0)
+			goto done;
+
+		if (value && strncmp(value, "union", 5) == 0) {
```

Comment:
Oh, I didn't think that was a valid option at the moment. git seems to support `ours`, `theirs`, and `union`. This could be updated to make sure the `union` were string-terminated.